conf/application.conf

```diff
@@ -172,6 +172,13 @@ mongo-async-driver {
   }
 }
 
+today-date-for-validation-flag {
+    is-2021 = false,
+    year = 2021,
+    month = 1,
+    day = 1
```

Comment:
and the date could be a simple string like `2021-01-01` and LocalDate can easily parse this string

Reply:
parsing can throw exceptions - not like this.....don't you think is better instead of handling parsing errors?